x/protocolpool/keeper/keeper.go

```diff
@@ -131,7 +135,7 @@ func (k Keeper) withdrawContinuousFund(ctx context.Context, recipientAddr string
 	}
 
 	if !toDistributeAmount.Equal(math.ZeroInt()) {
```

Comment:
Looking back I realized `WithdrawContinuousFund` will only loop through validators once when the first validator executes this msg (based on this condition).
Consider reverting `RecipientFundPercentageStatus` & `Endblocker` stuff since I just moved 1 loop from msg to enblocker. 
cc @facundomedica @tac0turtle